src/plugins/LADTable/components/LADTable.vue

```diff
@@ -84,6 +96,11 @@ export default {
             reorderPlan.forEach((reorderEvent) => {
                 this.$set(this.items, reorderEvent.newIndex, oldItems[reorderEvent.oldIndex]);
             });
+        },
+        metadataHasUnits(valueMetadatas) {
+            return valueMetadatas.filter((metadatum) => {
+                return metadatum.unit;
+            }).length;
```

Comment:
Now I'm just being super picky.... but the function name hasUnits implies a boolean return. You could have this be metadataWithUnits and your usage could be checking metadataWithUnits.length.

Also, if you wanted to be more terse you just need `metadatum => metadatum.unit`

Reply:
I like the terse route, good call, I also change up the implementation to check if the length is greater than 0, thus returning a boolean for more alignment between naming and functionality.